_includes/header.html

```diff
@@ -51,9 +51,9 @@
   {% endif %}
 		<div class="container">
       {% if page.url != "/" %}
-		    <nav class="navbar navbar-default navbar-static-top" role="navigation">
```

Comment:
It's recommended in the bootstrap docs for accessibility reasons, but it seems not to have any effects CSS wise. As this role addition is marked as warning by the W3 Validator, I guess newer browsers handle <nav> tags as navigation elements anyway.